src/promptflow/promptflow/_sdk/schemas/_flow.py

```diff
@@ -47,3 +43,21 @@ class FlowSchema(YamlFileSchema):
     description = fields.Str()
     display_name = fields.Str()
     tags = fields.Dict(keys=fields.Str(), values=fields.Str())
+
+
+class FlowSchema(BaseFlowSchema):
+    """Schema for flow dag."""
+
+    inputs = fields.Dict(keys=fields.Str(), values=NestedField(FlowInputSchema))
+    outputs = fields.Dict(keys=fields.Str(), values=NestedField(FlowOutputSchema))
+    nodes = fields.List(fields.Dict())
+    node_variants = fields.Dict(keys=fields.Str(), values=fields.Dict())
+
+
+class EagerFlowSchema(BaseFlowSchema):
```

Comment:
Shall we include the generated schema in the repo?

Reply:
Put eager flow and merged flow schema in here:
![image](https://github.com/microsoft/promptflow/assets/7776147/5241ac47-4c99-44c2-bfb0-ff88f6a88005)